### ISSUE TITLE
Additional files during create task causes error in --check process

### ISSUE THREAD

**Issue by dara2** [mjordan/islandora_workbench#476]
I am ingesting objects and I have 4 extra columns for different types of media (screenshot, transcript, thumbnail, and full_text). For each column header, I’ve added the column name and corresponding Media Use term ID in my config file. But when I run the --check, I get an error on the transcript column:
```
islandora@trislandora2-production:/opt/islandora_workbench$ ./workbench --config create_samplePROD.yml --check
OK, connection to Drupal at https://digitalcollections-i2-prod.tricolib.brynmawr.edu/ verified.
OK, configuration file has all required values (did not check for optional values).
OK, CSV file /mnt/ingest/PROD_TricoSampleMetadata.csv found.
OK, all 134 rows in the CSV file have the same number of columns as there are headers (51).
OK, URL aliases do not already exist.
Traceback (most recent call last):
  File "./workbench", line 1285, in <module>
    check_input(config, args)
  File "/opt/islandora_workbench/workbench_utils.py", line 1164, in check_input
    if field_definitions[csv_column_header]['handler'] == 'views':
KeyError: 'transcript'
islandora@trislandora2-production:/opt/islandora_workbench$
```
I tried removing all but the "screenshot" additional file type from my config file, because that had run just fine a month or so ago. But I get the same error as before (except KeyError is 'screenshot'). It seems not to like any of my additional files. Am I doing something incorrectly in my config file?

Config is here:
```
task: create
host: "https://digitalcollections-i2-prod.tricolib.brynmawr.edu/"
username: [myusername]
password: [mypassword]
input_dir: /mnt/ingest
input_csv: PROD_TricoSampleMetadata.csv
output_csv: PROD_TricoSampleMetadata-output.csv
output_csv_include_input_csv: true
allow_missing_files: true
allow_adding_terms: true
use_nid_in_media_title: 1
validate_title_length: false
strict_check: false
standalone_media_url: true
additional_files:
 - screenshot: 38
 - transcript: 40
 - thumbnail: 39
 - full_text: 228
 ```

CSV is attached.
I updated Workbench and Python libraries today (and was seeing the same error before updating).

[PROD_TricoSampleMetadata.csv](https://github.com/mjordan/islandora_workbench/files/9708165/PROD_TricoSampleMetadata.csv)

**Comment by mjordan** [comment]
What is the 'transcript' field's field type? Entity reference, text, etc.

**Comment by dara2** [comment]
It's not a field type, it's just a custom Islandora Media Use term that we're uploading media for.

**Comment by mjordan** [comment]
Ah, of course, sorry. I see that in your "additional_files" configuration. This is a bug that I'll need to fix. I'll try to get to it tonight.

**Comment by dara2** [comment]
Awesome - thanks, Mark!

**Comment by mjordan** [comment]
@dara2 can you pull in updates to the main branch and try again?

**Comment by dara2** [comment]
This worked great - thank you for the quick fix!

**Comment by mjordan** [comment]
Closing, resolved with d4f8a251c9468bd4dea8720fbe43cddb07842280.
